pkg/translate/translate_value.go

```diff
@@ -535,7 +542,7 @@ func (t *ReverseTranslator) isEnablementPath(path util.Path) bool {
 
 	pstr := path.String()
 	if pstr == "mixer.policy.enabled" || pstr == "mixer.telemetry.enabled" ||
-		pstr == "gateways.enabled" ||
+		pstr == "gateways.enabled" || pstr == "mixer.enabled" ||
```

Comment:
there are enough of these that you might want to define as a map at the top of the file. it will be hard to keep it updated here.

Reply:
added a var at the top of the file